.github/workflows/ci.yaml

```diff
@@ -56,7 +56,7 @@ jobs:
 
   tlint:
     name: Lint checks
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
why is this needed?

Reply:
No strict need detected actually, I guess I just pinned os version for all the jobs.
I'll revert that to `ubuntu-latest` if it is preferable.

Reply:
If you don't need to do so, I suggest dropping the commit.
This might be needed if we decide to go with a matrix testing and test against an OTP version that is not supported by ubuntu 22.04, we can re-add modification in that PR if needed (there is already a debt for this).

Reply:
Makes sense, thanks! Changed back to latest